### ISSUE TITLE
Book 05: Sentence 091

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#26]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/3342608d-4f3f-4011-bf6b-c9809dd62ace)

```
# sent_id = Book_05_-_The_Omens.txt:91
# text = No yoan, netetzaujloia moteittitiaia: in jtoca cujtlapanton, anoço cintanaton: mjtoa in acolhoacan centlapachton: iuh tocaiotilo.
# text[norm] = No ihuan, netetzahuiloya moteittitiaya: in itoca cuitlapanton, anozo cintanaton: mihtoa in Acolhuacan centlapachton: iuh tocayotilo.
# text[orig] = No·yoan·,·netetzaujloia·mo¶teittitiaia·:·injtoca·cujtla·panton·,·anoço·cintanaton·:·mjtoa·inacolho¶acan·centlapachton·:·iuh·tocaiotilo·.
# text[eng] = With that also, they considered an omen appeared before them: its name "[...]", or "[...]": it is said in Acolhuacan "[...]": like this they were named.
# text[osp] = Auja otra manera de fantasma, que de noche aparecia: ordinariamente en los lugares, donde yvan a hazer sus necesidades de noche: si alli, les aparecia vna muger pequeña enana, que la llamauan cujtlapanton, o por otro nombre centlapachton.
# references = A&D/179:2
1	No	no	ADV	_		4	advmod	4:advmod	Folio=12v|Paragraph=4|Line=1|Norm=No|Gloss=también|Analysed=Yes
2	yoan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	4	obl	4:obl	Folio=12v|Paragraph=4|Line=1|Norm=ihuan|Gloss=con|Analysed=Yes
3	,	,	PUNCT	_		4	punct	4:punct	Folio=12v|Paragraph=4|Line=1|Norm=,|Analysed=Yes
4	netetzaujloia	tetzahuia	VERB	_	_	0	root	0:root	Folio=12v|Paragraph=4|Line=1|Norm=netetzahuiloya
5	moteittitiaia	ittitia	VERB	_	Animacy[iobj]=Anim|Aspect=Imp|Number[subj]=Plur,Sing|Person[iobj]=3|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Ditr|Tense=Past|VerbForm=Fin	4	csubj	4:csubj	Orig=mo¶·teittitiaia|Folio=12v,12v|Paragraph=4,4|Line=1,2|Norm=moteittitiaya|Analysed=Yes
6	:	:	PUNCT	_		8	punct	8:punct	Folio=12v|Paragraph=4|Line=2|Norm=:|Analysed=Yes
7	in	in	SCONJ	_	_	8	mark	8:mark	Orig=injtoca|Folio=12v|Paragraph=4|Line=2|Norm=in
8	jtoca	toca	NOUN	_	_	5	acl	5:acl	Orig=injtoca|Folio=12v|Paragraph=4|Line=2|Norm=itoca|Gloss=su.nombre
9	cujtlapanton	cuitlapantli	NOUN	_	NounDeriv=Dim	8	nmod	8:nmod	Orig=cujtla·panton|Folio=12v,12v|Paragraph=4,4|Line=2,2|Norm=cuitlapanton|Analysed=Yes
10	,	,	PUNCT	_		12	punct	12:punct	Folio=12v|Paragraph=4|Line=2|Norm=,|Analysed=Yes
11	anoço	anozo	ADV	_		12	cc	12:cc	Folio=12v|Paragraph=4|Line=3|Norm=anozo|Analysed=Yes
12	cintanaton	cintanaton	NOUN	_	State=Abs	9	conj	9:conj	Folio=12v|Paragraph=4|Line=3|Norm=cintanaton|Analysed=Yes
13	:	:	PUNCT	_		14	punct	14:punct	Folio=12v|Paragraph=4|Line=3|Norm=:|Analysed=Yes
14	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	4	parataxis	4:parataxis	Folio=12v|Paragraph=4|Line=3|Norm=mihtoa|Analysed=Yes
15	in	in	SCONJ	_	_	16	mark	16:mark	Orig=inacolho¶·acan|Folio=12v,12v|Paragraph=4,4|Line=3,4|Norm=in
16	acolhoacan	Acolhuacan	PROPN	_		14	parataxis	14:parataxis	Orig=inacolho¶·acan|Folio=12v,12v|Paragraph=4,4|Line=3,4|Norm=Acolhuacan|Analysed=Yes
17	centlapachton	centlapachton	NOUN	_	State=Abs	16	orphan	16:orphan	Folio=12v|Paragraph=4|Line=4|Norm=centlapachton|Analysed=Yes
18	:	:	PUNCT	_		20	punct	20:punct	Folio=12v|Paragraph=4|Line=4|Norm=:|Analysed=Yes
19	iuh	iuh	ADV	_		20	advmod	20:advmod	Folio=12v|Paragraph=4|Line=4|Norm=iuh|AmbigNorm=True|Override=False|Gloss=así|Analysed=Yes
20	tocaiotilo	tocayotia	VERB	_	_	4	parataxis	4:parataxis	Folio=12v|Paragraph=4|Line=4|Norm=tocayotilo|Ambiguous=Yes|Analysed=Yes
21	.	.	PUNCT	_		4	punct	4:punct	Folio=12v|Paragraph=4|Line=4|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
López Austin has:
> Y también se tenía por augurio si aparecía la llamada "espaldilla" "cestillo de mazorcas". En Acolhuacan se llama "la enteramente aplastada". Así era llamada.

Questions:
- What is the relation between *netetzaujloia* and *moteittitiaia*, is it `csubj`, if so, why no *in*? 
- Is *anozo* a `CONJ` here? 
- What is going on with *mihtoa in Acolhuacan centlapachton*?

**Comment by Lguyogiro** [comment]
> Is anozo a CONJ here?

I am in favor of anozo being cconj

**Comment by Lguyogiro** [comment]
> What is the relation between netetzaujloia and moteittitiaia, is it csubj, if so, why no in?

Yeah it is weird with no `in`. We could say, because of no subordinator there is `parataxis`? I don't think we've assigned csubj to anything yet without `in`.

**Comment by Lguyogiro** [comment]
> What is going on with mihtoa in Acolhuacan centlapachton?

I don't know how to analyze this, specifically the _in_ before _Acolhuacan_.  One maybe weird option could be "It is said (that) in Acolhuacan, centlapachton is how it is named."

**Comment by ftyers** [comment]
> > What is going on with mihtoa in Acolhuacan centlapachton?
> 
> I don't know how to analyze this, specifically the _in_ before _Acolhuacan_. One maybe weird option could be "It is said (that) in Acolhuacan, centlapachton is how it is named."

Like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/6ca365b3-7be0-4092-9712-d535e82150a0)

**Comment by Lguyogiro** [comment]
Yeah that looks good

**Comment by ftyers** [comment]
Merged in 2a3aff3.
